content/docs/common-areas/cdn.md

```diff
@@ -0,0 +1,20 @@
+---
+title: CDN (Content Delivery Network)
+weight: 6
+description: A content delivery network, or Content Distribution Network (CDN), is a group of globally dispersed servers that accelerates the delivery of web content by bringing it closer to the user. CDNs have the benefits of high performance in loading content, minimizing bandwidth costs, and reaching content globally. CDN caches content on proxy servers that are located near where the user is, reducing latency.
+---
+
+# CDN (Content Delivery Network)
+
+A content delivery network, or Content Distribution Network (CDN), is a group of globally dispersed servers that accelerates the delivery of web content by bringing it closer to the user. CDNs have the benefits of high performance in loading content, minimizing bandwidth costs, and reaching content globally. CDN caches content on proxy servers that are located near where the user is, reducing latency.
+
+The main goal of a CDN is to quickly and securely provide any kind of digital content to users, wherever they are in the world and with whatever device they prefer. In order to improve speed and connectivity, a CDN places servers at the exchange points between different networks. Depending on the desired level of coverage, the majority of CDN providers will offer their services over a variety of clearly defined PoPs. These PoPs are referred to as "edges", "edge nodes", "edge servers", or "edge networks".
+
+A CDN stores cached content on edge servers in point-of-presence (PoP) locations that are close to end users to minimize latency. Depending on the architecture, different CDNs can have thousands of servers spread across numerous distant points of presence (PoPs). Others create a worldwide network with a sparse distribution of PoPs.
+
+Even though the steps depend on the CDN provider, the common steps are:
+
+- Analyze the website and use a tool such as CDN Finder to check whether a CDN is integrated into the website.
+- Enable the CDN by selecting the preferred CDN service.
+- Configure the settings of your CDN service.
+- Update your DNS – update your DNS records to point to your CDN service.
```

Comment:
@Yemaneberhan-Lemma, can you please remove this so that I can merge it? I had raised a comment on this in a previous PR but I don't see this changed or removed.